UPGRADE.md

```diff
@@ -1,5 +1,10 @@
 # Upgrade to 4.0
 
+## BC BREAK: Userland lazy objects are no longer supported
+
+Userland lazy objects are no longer supported. Native lazy objects are used
+instead.
```

Comment:
Let's please link the PHP RFC

Reply:
or simply [the docs](https://www.php.net/manual/en/language.oop5.lazy-objects.php)?